composer.json

```
@@ -21,7 +21,7 @@
     "10up/phpcs-composer": "dev-master"
   },
   "scripts": {
-    "lint": "phpcs .",
+    "lint": "phpcs . --runtime-set testVersion 7.2-",
```

```suggestion
    "lint": "phpcs . --runtime-set testVersion 7.0-",
```

7.0 according to readme.txt